include/z64pause.h

```diff
@@ -141,6 +141,11 @@ typedef enum WorldMapPointState {
     /* 2 */ WORLD_MAP_POINT_STATE_HIGHLIGHT
 } WorldMapPointState;
 
+// Values for respectively PauseContext.pagesYOrigin1 and R_PAUSE_PAGES_Y_ORIGIN_2
+// so that pause pages rotate around their lower edge instead of the middle.
```

Comment:
having a hard time parsing what this is trying to say. "values for x so that y" doesnt read right to me. is there something more to add about the values that achieves that goal or something?

or maybe 
```
// Values for PauseContext.pagesYOrigin1 and R_PAUSE_PAGES_Y_ORIGIN_2
// respectively that allow the pause pages to rotate around their lower edge instead of the middle.
```
?

Reply:
is this better? 6cbc21c47c40e1950d8496001e119087b4930ece

Reply:
yep thanks, meging